tests/FormaterTests.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+import os
+from app.Notebook import Cell, Notebook
+from app.Formater import ToNotebook, ToPy
+
+
+class FormaterTests(unittest.TestCase):
+    def test_topy_class_creation(self):
+        topy = ToPy()
+        self.assertIsInstance(topy, ToPy)
+
+    def test_to_notebook_class_creation(self):
+        tonb = ToNotebook()
+        self.assertIsInstance(tonb, ToNotebook)
+
```

Comment:
The ToPy and ToNotebook should probably be separate classes, probably with a bass class as a lot of testcases are duplicates, the only difference is the class.